### ISSUE TITLE
Difficult to find extension in Marketplace if you don't know it's name

### ISSUE THREAD

**Issue by DanStevens** [stkb/Rewrap#236]
I'd not used this extension in a long time and I couldn't remember it's name. I tried searching for 'word wrap' and 'hard wrap' and neither search brought this extension up in the result. I had to use Google search to find out the exact name and then it would show in the results.

**Comment by JKAnderson409** [comment]
I agree. It had become almost a hobby for me to look for an extension like this before I found it today. It's very good at what it does and I think it could be much more popular with a name that includes the word "column", like "Column Wrap" or something like that.

Great work on this Steve and other contributors, having auto-wrap at 80 columns is pure zen.

**Comment by DanStevens** [comment]
I wouldn't suggest changing the name except as a last resort. Does the VS Code Marketplace not have ways to tweak search results e.g. keywords? Or may be the description can be improved?

**Comment by stkb** [comment]
It seems that a lot of new extensions have been added to the marketplace since I last looked at this, and I agree placement in the search results wasn't great anymore.

I don't want to change the name, but for the latest release, I've changed the keywords a bit. So they are now: "word wrap", "hard wrap", "comments", "column", "line length", "fill", "paragraph". I've also changed the description to "Hard word wrapping for comments and other text at a given column." (I didn't want to include the owrd *automatic* as (for now) you still need to trigger it manually).

Trying again after those changes, the results are still not really good. If you search for "word wrap", there are a lot of extensions with "word" in the name that come up first. I don't know if it needs some time to re-index or if this extension appears lower in the results because it hasn't had much activity lately. Time will tell a little.
